PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,12 @@
+Hello!
+
+Thank you for choosing to apply for am IOOS project!
+You can use pull requests (PR) on this repository to add on a draft of your proposal before the student application period begins to get some feedback.
+
+Before you open a PR for your proposal please make sure that you have contacted our organization first.
+You can find as at ?????
```

Comment:
Do we want to add an e-mail here or should we use discussion/issues for this?
We don't have an open slack or something similar for faster communications and maybe we want to reserve that only for accepted students.

```suggestion
You can contact us using https://github.com/ioos/gsoc/issues or via your PR.
```

Reply:
Sorry for jumping in, but having worn the shoes of a newbie contributor before, I'd suggest having an open discussion/forum is more helpful than a private mailbox. The reason being, we learn a lot more from the questions of other contributors and might save time than back-and-forth email convos. GitHub Issue/Discussions would be good, though a Google Groups/mailing list could also do.

Reply:
@ayushanand18 your feedback is is extremely Valuable! Thank you for jumping in!

I'm not sure we can handle a Google group. Maybe GitHub discussion is the best way forward? What do you think?

Reply:
I'm still having trouble delineating between Discussions and Issues. To me they are one in the same and Discussions doesn't seem to foster more conversation than Issues, at least from what I've seen. In the end, I agree with @ayushanand18 and recommend we use the GitHub platform for these conversations.

Reply:
> To me they are one in the same and Discussions doesn't seem to foster more conversation than Issues

Same for me. The only advantage is to create a separation of context for big projects (issues are actionable and discussions will become issues when settled). Maybe we don't need it here and issues are fine.

Reply:
Great convo here, I second with @MathewBiddle. Issues are more beginner friendly and more popular, because most new contributors aren't much aware of GitHub and all of its features. 
I believe discussions over GitHub Issues against each project on this repo should be fine and separate from actionable items (which should be on the project's repo itself).